libretro/libretro.cpp

```diff
@@ -888,11 +888,16 @@ float System_GetPropertyFloat(SystemProperty prop)
    {
       case SYSPROP_DISPLAY_REFRESH_RATE:
          return 60.f;
+      case SYSPROP_DISPLAY_SAFE_INSET_LEFT:
+      case SYSPROP_DISPLAY_SAFE_INSET_RIGHT:
+      case SYSPROP_DISPLAY_SAFE_INSET_TOP:
+      case SYSPROP_DISPLAY_SAFE_INSET_BOTTOM:
+         return 0.0f;
       default:
          break;
    }
 
-   return -1;
+   return 0.0f;
```

Comment:
I think we should stick to -1 here for now.

Reply:
allright.